pkg/monitor_agent/agent.go

```diff
@@ -16,7 +17,8 @@ import (
 )
 
 var (
-	logger = logf.Log.WithName("monitor_agent")
+	logger                     = logf.Log.WithName("monitor_agent")
+	octarineValidatingWebhooks = map[string]string{"octarine-guardrails": "octarine-guardrails"}
```

Comment:
this will not work if the user gave another name to the Octarine CR (try changing the name in the CR to shani for example, then the webhook will be named shani-guardrails). You can use the guardrailsWebhookName() method in names.go

Reply:
It looks like I can't import the guardrailsWebhookName method from the names.go file because it will create an import cycle (monitor_agent imports from octarine package, which names.go is from and octarine package imports from monitor_agent under monitor_reconciler). 
Is there another way to find the octarine name?
Adding it like we added the Version to the octarineSpec will work?